File: ios/Classes/data/PhoneNumberType.swift
//
//  PhoneNumberType.swift
//  contacts
//
//  Created by Maurits van Beusekom on 28/08/2018.
//

import Foundation

enum PhoneNumberType : String, Codable {
    case home = "home"
    case homeFax = "homeFax"
    case iphone = "iphone"
    case main = "main"
    case mobile = "mobile"
    case work = "work"
    case workFax = "workFax"
    case pager = "pager"
    case other = "other"
}

File: android/src/main/kotlin/com/baseflow/contactsplugin/data/PhoneNumberType.kt
package com.baseflow.contactsplugin.data

import com.google.gson.annotations.SerializedName

enum class PhoneNumberType {
    @SerializedName("home")
    HOME,
    @SerializedName("homeFax")
    HOME_FAX,
    @SerializedName("iphone")
    IPHONE,
    @SerializedName("main")
    MAIN,
    @SerializedName("mobile")
    MOBILE,
    @SerializedName("pager")
    PAGER,
    @SerializedName("work")
    WORK,
    @SerializedName("workFax")
    WORK_FAX,
    @SerializedName("other")
    OTHER,
}